docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,339 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
+
+```
+ziti edge update config test-host-config host.v2 --data '
+{
+    "terminators" : [
+        {
+            "address": "192.168.3.136",
+            "port" : 8080,
+            "protocol": "tcp"
+        },
+        {
+            "address": "192.168.3.137",
+            "port" : 8080,
+            "protocol": "tcp"
+        }
+    ]
+}
+'
+```
+
+Now each ER/T will create two terminators, one for each endpoint, for a total of four terminators. Now that we have
+multiple endpoints we'll want to know when they are healthy or unavailable we can use the just the endpoints which
+are working. We can accomplish this by adding health checks to the configuration.
+
+```
+ziti edge update config test-host-config host.v2 --data '
+{
+    "terminators" : [
+        {
+            "address": "192.168.3.136",
+            "port" : 8080,
+            "protocol": "tcp",
+            "portChecks" : [
+                {
+                    "address" : "192.168.3.136:8080",
+                    "interval" : "5s",
+                    "timeout" : "100ms",
+                    "actions" : [
+                        {
+                            "trigger" : "fail",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark unhealthy"
+                        },
+                        {
+                            "trigger" : "pass",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark healthy"
+                        }
+                ]
+            }
+            ]
+        },
+        {
+            "address": "192.168.3.137",
+            "port" : 8080,
+            "protocol": "tcp",
+            "portChecks" : [
+                {
+                    "address" : "192.168.3.137:8080",
+                    "interval" : "5s",
+                    "timeout" : "100ms",
+                    "actions" : [
+                        {
+                            "trigger" : "fail",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark unhealthy"
+                        },
+                        {
+                            "trigger" : "pass",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark healthy"
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+}
+'
+```
+
+Our configuration has gotten quite large! However, we've gained a good bit of functionality with our new additions.
+Our servers will now be pinged every five seconds. If a the health check fails three times in a row, the associated
+terminator will be marked unhealthy, which means its precedence will be set to `failed`. If subsequently the health check
+passes three times in a row, its precedence will be reset to its original value.
+
+This example uses simple port checks, but http checks are also supported. The checks are per-terminator, so if the
```

Comment:
How can one use health checks with a CIDR range? Or can one? I expect not? or one just pick "any server" in the CIDR range and call it good enough? I'd like to see a little bit about health checks and CIDR. Also mention FQDN here. Does heath check need to be IP? I suspect not... but let's doc it